app/src/androidTest/java/com/github/sdp/ratemyepfl/database/fakes/FakeReviewsRepository.kt

```diff
@@ -122,38 +124,9 @@ class FakeReviewsRepository @Inject constructor() : ReviewRepository {
         reviewList[0].dislikers = listOf(FAKE_UID_3, FAKE_UID_4)
     }
 
-    override suspend fun take(number: Long): QuerySnapshot =
-        Mockito.mock(QuerySnapshot::class.java)
-
-    override suspend fun getById(id: String): DocumentSnapshot =
-        Mockito.mock(DocumentSnapshot::class.java)
-
-    override fun remove(id: String): Task<Void> {
```

Comment:
You should keep this function here, or the tests won't work.

Reply:
I guess it's one of the methods that is implemented in the FakeRepository. Each fake now inherits it (it is much simpler to maintain).

Reply:
Yes but this function actually does something here. In the FakeRepository you just mock.

Reply:
I am talking about the remove function.